src/patternfly/components/Page/page.scss

```diff
@@ -279,9 +279,7 @@
 .pf-c-page__header-tools {
   grid-column: 2 / 3;
   align-items: center;
-  margin-top: var(--pf-c-page__header-tools--MarginTop);
```

Comment:
This is still defined at https://github.com/patternfly/patternfly-next/blob/829b5f6b40d992988b78482ab11aa580a326e684/src/patternfly/components/Page/page.scss#L61